webapp/crashstats/authentication/admin.py

```diff
@@ -38,12 +38,12 @@ class UserAdminBetter(UserAdmin):
         "get_policyexception",
     ]
 
+    @admin.display(description="Hacker (PII)")
```

Comment:
Instead of setting properties on the methods, there's a decorator that you can use to set things. We switched to that.